app/src/main/java/com/vasilevkin/catsanddogs/MainApplication.kt

```diff
@@ -0,0 +1,41 @@
+package com.vasilevkin.catsanddogs
+
+import android.app.Application
+import com.facebook.stetho.Stetho
+import com.vasilevkin.catsanddogs.di.animalListModule
+import com.vasilevkin.catsanddogs.di.splashModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.android.ext.koin.androidLogger
+import org.koin.core.context.startKoin
+
+class MainApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        // Start Koin
+        startKoin {
+            androidContext(this@MainApplication)
+            androidLogger()
+            modules(listOf(animalListModule, splashModule))
+        }
+
+        // Create an InitializerBuilder
+        val initializerBuilder = Stetho.newInitializerBuilder(this)
```

Comment:
you could write it like - Stetho.apply { ... }

Reply:
refactored using .also {}